src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -645,8 +649,30 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
     // If the theme asks for Mica, then drop out our background, so that we
     // can have mica too.
     void MainPage::_UpdateBackgroundForMica()
-
     {
+        bool isMicaAvailable = false;
+
+        // Check to see if our hosting window supports Mica at all. We'll check
+        // to see if the window has Mica enabled - if it does, then we can
+        // assume that it supports Mica.
+        //
+        // We're doing this instead of checking if we're on Windows build 22621
+        // or higher.
+        if (_hostingHwnd.has_value())
+        {
+            int attribute = DWMSBT_NONE;
+            const auto hr = DwmGetWindowAttribute(*_hostingHwnd, DWMWA_SYSTEMBACKDROP_TYPE, &attribute, sizeof(attribute));
+            if (SUCCEEDED(hr))
+            {
+                isMicaAvailable = attribute == DWMSBT_MAINWINDOW;
+            }
+        }
+
+        if (!isMicaAvailable)
+        {
+            return;

```

Comment:
now, does this hot reload if you turn mica off? ;)

Reply:
Sure seemed like it did